frontend/public/components/factory/list-page.jsx

```diff
@@ -284,7 +285,8 @@ export const FireMan_ = connect(null, { filterList })(
                 />
               </div>
             )}
-            {!title && badge && (
+            {/* Show the badge in the filter bar if `filterBarBadge` and there's a no title. */}
```

Comment:
Nit: not sure this comment is needed. The code is pretty explicit in this description as well.

If anything, `filterBarBadge` is a confusing prop name... but with a brief context understanding I think the name makes sense too...

Reply:
Yeah, I struggled with the name. Let me know if you a suggestion. Names are hard :/

Reply:
Yeah :/ Naming can be quite difficult at times. I definitely don't think we should relate it to `search` as this component doesn't know anything about where it is used.

I wonder if it's as simple as `noTitleBadge` or something like that? Naming is hard, but I think if it's specific to "I don't have a title, but I still want the badge"?

Reply:
Honestly, it feels like the pages that want to render the badge themselves should opt-out instead of having the search page opt-in. The default should be to show the badge as part of the list component. Opinion?

Reply:
An interesting concept. My only concern is this check explicitly looks for a title before validating if it shows a badge. We'd need to flip that on it's head.

I see default logic as a "path of least resistance" and opt-out props as a way to get around that.

So I guess the logic would need to be:

- No badge, show nothing
- If I have a badge, show it in the filter area
- If I have a badge & title, show the badge in the title area instead of the filter area?

Would that cover the flow of the logic?

Reply:
Ah, our project pages don't show a title, it's a custom block... Hmm, perhaps flipping it is the right thing to do. We are the exception to the flow of things.

Reply:
@spadgett I think you're onto something. If our project pages say `hideBadge` instead of the search page saying show in filter area... I think we land on a smoother solution.